### PR TITLE
Prepare release v1.19.0

### PR DESCRIPTION
**Pull request by sfc-gh-aalam**
1. Which Jira issue is this PR addressing? Make sure that there is an accompanying issue to your PR.

   

   Fixes SNOW-NNNNNNN

2. Fill out the following pre-review checklist:

   - [ ] I am adding a new automated test(s) to verify correctness of my new code
      - [ ] If this test skips Local Testing mode, I'm requesting review from @snowflakedb/local-testing
   - [ ] I am adding new logging messages
   - [ ] I am adding a new telemetry message
   - [ ] I am adding new credentials
   - [ ] I am adding a new dependency
   - [ ] If this is a new feature/behavior, I'm adding the Local Testing parity changes.

3. Please describe how your code solves the related issue.

   Please write a short description of how your code change solves the related issue.

### CODE REVIEW

**Review by sfc-gh-nkrishna — APPROVED** [review]

**Review by sfc-gh-jdu — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -62,7 +62,7 @@
 - Added support for named aggregations in `DataFrame.aggregate` and `Series.aggregate` with `axis=0`.
 - `pd.read_csv` reads using the native pandas CSV parser, then uploads data to snowflake using parquet. This enables most of the parameters supported by `read_csv` including date parsing and numeric conversions. Uploading via parquet is roughly twice as fast as uploading via CSV.
 - Initial work to support an Index directly in Snowpark pandas. Support for Index as a first-class component of Snowpark pandas is coming soon.
-- Added lazy index constructor and support for `len`, `shape`, `size`, `empty`, `to_pandas()` and `names`. For `df.index`, Snowpark pandas creates a lazy index object. 
+- Added lazy index constructor and support for `len`, `shape`, `size`, `empty`, `to_pandas()` and `names`. For `df.index`, Snowpark pandas creates a lazy index object.
```
Comment (sfc-gh-mheavin) [inline CHANGELOG.md:65]: Added a lazy

File: CHANGELOG.md [hunk 2]
```diff
@@ -62,7 +62,7 @@
 - Added support for named aggregations in `DataFrame.aggregate` and `Series.aggregate` with `axis=0`.
 - `pd.read_csv` reads using the native pandas CSV parser, then uploads data to snowflake using parquet. This enables most of the parameters supported by `read_csv` including date parsing and numeric conversions. Uploading via parquet is roughly twice as fast as uploading via CSV.
 - Initial work to support an Index directly in Snowpark pandas. Support for Index as a first-class component of Snowpark pandas is coming soon.
```
Comment (sfc-gh-mheavin) [inline CHANGELOG.md:64]: Why is Index capitalized?

**Review by sfc-gh-mheavin — COMMENTED** [review]
Some small suggestions.

File: CHANGELOG.md (hunk 2 above)
Comment (sfc-gh-aalam) [reply CHANGELOG.md:64]: @sfc-gh-nkrishna would you have more context here? would adding `pd.Index` make more sense here?

**Review by sfc-gh-mheavin — APPROVED** [review]
Thanks for the updates

**Pull request merged**